sklearn/neighbors/_graph.py

```diff
@@ -159,7 +160,8 @@ def radius_neighbors_graph(X, radius, *, mode='connectivity',
         for mode='distance'.
 
     n_jobs : int, default=None
-        The number of parallel jobs to run for neighbors search.
+        The number of jobs to run in parallel; the k-nearest neighbor
+        computations are performed parallel to each other.
```

Comment:
I think the original is clear enough about what `n_jobs` is used for.

Aside: `radius_neighbors_graph` finds the neighbors in a given radius.

Reply:
Thanks for the feedback! We can leave it as it is then. Also thanks for the clarification.